libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -927,6 +952,26 @@ instance ToJSON TeamDeleteData where
       [ "password" .= _tdAuthPassword tdd
       ]
 
+instance ToJSON LimitedTeamSize where
```

Comment:
don't forget to add a line to `libs/brig-types/test/unit/Test/Brig/Types/Common.hs`

Reply:
Done.